### ISSUE TITLE
Accessibility Issues in UI Components

### ISSUE THREAD

**Issue by suryaprakash666** [suryaprakash666/animythik_frontend#10]
Many UI components lack proper ARIA attributes and might have color contrast issues, particularly in dashboard components. This affects users with disabilities.

**Comment by kotlalokeshwari098** [comment]
Hi @suryaprakash666 , I am interested to solve this issue!

**Comment by suryaprakash666** [comment]
i want to know a bit about you, and how you get to know about this project?

**Comment by kotlalokeshwari098** [comment]
Hi @suryaprakash666 ,I came across it while browsing GitHub issues related to accessibility, and I noticed the one about missing ARIA attributes and color contrast issues in the dashboard components.

**Comment by suryaprakash666** [comment]
Actually this is a live project being built for as product of a company, so we cannot give you direct access to the project but we appreciate your interest and contact you further when positions opened for this role.
Keep connection with us.

**Comment by kotlalokeshwari098** [comment]
Thanks for the response — I completely understand and respect that the project is part of an active product build.

I’d be happy to stay connected and would definitely be interested in exploring opportunities with your team if and when they open up.

Place: Telangana ,India
Current : 3rd-year Computer Science Engineering student

I’m currently focusing on full-stack development and I enjoy contributing to projects.
